data/blog/electron-3-0.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Electron 3.0.0
+author: codebytere
+date: '2018-09-20'
+---
+
+The Electron team is excited to annouce that the first stable release of Electron `v3.0.0` is now
+available from [electronjs.org](https://electronjs.org/)! It's jam-packed with upgrades, fixes, and new features, and we can't wait to see what you build with them. Below are more details about the specifics of this latest release, and we welcome your feedback as you explore.
+
+---
+
+## Release Process
+
+As we undertook development of `v3.0.0`, we sought to more empirically define criteria for a stable release by formalizing the feedback progress for progressive beta releases. This feedback process involved a number of our consumers in Electron's ecosystem, including but not limited to: Slack, VSCode, Atom, OpenFin, Symphony, and Oculus. We wouldn't have been able to mark this milestone without their work to let us know about the blockers and needs they faced with each successive beta as they journeyed down the road to `v3.0.0` upgrades for their apps.
```

Comment:
Should we include a statement for those interested in participating in future cycles of the App Feedback Program to email info@electronjs.org?

Reply:
definitely ☝️